extensions/graphviz_support/setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+setup(
+    name="graphviz_support",
+    version="1.0.0",
+    description="LabGraph Monitor Improvement - Graphviz for LabGraph graphs",
+    packages=find_packages(),
+    python_requires=">=3.6",
+    install_requires=[
+        "labgraph==1.0.2",
```

Comment:
Ideally if we could also support labgraph 2.0

Reply:
Done